ui/src/main/java/org/mini2Dx/ui/UiContainer.java

```diff
@@ -700,6 +700,9 @@ private boolean handleTextInputKeyUp(int keycode) {
 		case Input.Keys.LEFT:
 			activeTextInput.moveCursorLeft();
 			break;
+		case Input.Keys.ESCAPE:
+			unsetActiveAction();
+			break;
```

Comment:
Lines 680 and 681 require to press enter twice, for the very first time, to exit text input... Is that intended? If so, why?